src/components/shared/Form/Form.jsx

```diff
@@ -1,4 +1,6 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
@Akinmyde  any reason for disabling this rule?

Reply:
> @Akinmyde any reason for disabling this rule?

I disabled it because I don't need to use 'to' in the 'Link' tag.